docs/libcurl/curl_url_cleanup.3

```diff
@@ -27,7 +27,7 @@ curl_url_cleanup - free a CURLU handle
 #include <curl/curl.h>
 
 void curl_url_cleanup(CURLU *handle);
-.fi
+.
```

Comment:
1. My mandoc doesn't complain on this
2. This pattern is used in all libcurl function man pages, why are these three singled out?
3. Why leave a dot?

Reply:
Hmm, I also don´t get these anymore, apparently mandoc stopped complaining about it sometime ago.

This hunk can be dropped.

I don't know why the original submitter decided to go with a dot, I wasn't the original reviewer of the patch either so I just kept refreshing it. I figured sending it upstream was a good opportunity to drop changes like this.